Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -273,6 +274,15 @@ def main():
     pr.create_issue_comment(body)
     print(f'{t.cyan}Created welcome comment{t.normal}')
 
+    print('contributors.md section')
+    print(f'pack path: {pr_files[0]}')
+    ver = get_pack_metadata(pr_files[0]).get('currentVersion')
```

Comment:
no, it's not the pack metadata file.
check the get_pack_metadata implementation.

It gets any file path from the pack - it can be readme, metadata or any other file and returns the metatdata.